Sources/Public/Animation/AnimationView.swift

```diff
@@ -123,10 +125,24 @@ final public class AnimationView: AnimationViewBase {
   /**
    Describes the behavior of an AnimationView when the app is moved to the background.
 
-   The default is `pause` which pauses the animation when the application moves to
-   the background. The completion block is called with `false` for completed.
+   The default for the Main Thread animation engine is `pause`,
+   which pauses the animation when the application moves to
+   the background. This prevents the animation from consuming CPU
+   resources when not on-screen. The completion block is called with
+   `false` for completed.
+
+   The default for the Core Animation engine is `doNothing`, since
+   the Core Animation engine does not have any CPU overhead.
```

Comment:
Does this mean that the animation will start up again automatically? Or is the implicit behavior of core animation to pause when offscreen (from what I remember that's the case?), and maybe we shouldn't have the `doNothing` case and instead just handle the `pause` state like:
```swift
case .pause:
  switch configuration.renderingEngine {
  case .main:
    removeCurrentAnimation()
  case .coreAnimation:
    break
  }
```

Reply:
The default behavior of Core Animation is to remove the animation when the app is backgrounded. That doesn't work for the new engine, since the view is _only_ rendered via the animations (including when paused), so we set `isRemovedOnCompletion = false` on all of the animations. By default for animations playing in realtime, this causes the animation to appear as it it was playing continuously while the app was in the background (e.g. the layer's "current time" continues to advance).

This is semantically different from all of the other existing background modes, so I think we need a new option for it.

Reply:
Got it, makes a lot of sense, thanks! Some of this discussion would be helpful to have in the documentation comments I think

Reply:
Updated and added more extensive docs in https://github.com/airbnb/lottie-ios/pull/1520/commits/0dec30ff550f9dbae219f4b5c6c488fed9f9fb7c